FRAuth/FRAuth/Callbacks/WebAuthnCallback.swift

```diff
@@ -14,7 +14,8 @@ import Foundation
 
 /// WebAuthnCallback represents AM's WebAuthn MetadataCallback, and is a parent class of WebAuthnRegistrationCallback and WebAuthnAuthenticationCallback
 open class WebAuthnCallback: MetadataCallback {
-    
+    /// Boolean indicator whether or not if response supports JSON response
```

Comment:
This comment is somewhat off. Should this be:
_"Boolean flag indicating whether the response supports JSON."_

Reply:
I blame the AI :D I fixed it